render.go

```diff
@@ -39,9 +39,21 @@ type Renderer struct {
 	// Defaults to empty destinations for all hashtags.
 	Resolver Resolver
 
+	Attributes []Attribute
+
 	hasDest sync.Map // *Node => struct{}
 }
 
+// Attribute defines an attribute to be added to an HTML tag.
+//
+// Attribute{ Attr: "class", Value: "tag"}
+//
```

Comment:
In godoc, to have this render as a code block, we have to use at least two spaces or a tab.

```suggestion
//
//	Attribute{ Attr: "class", Value: "tag"}
//
```